### PR TITLE
Document change to update known issue.

### PR DESCRIPTION
**Pull request by lipingxue**
**What this PR does / why we need it**:
This change is  to update 
1. known issue:  document the issue when specify VC using IPv6 address in "vsphere-csi.conf" when deploying the plugin and the workaround to this issue
2. In TKGm env,  user need to exclude the resources in the block list for backup/restore to avoid "Partially Failed" error.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/848/

File: docs/known_issues.md [hunk 1]
```diff
@@ -64,3 +64,24 @@ Users might require ways to provide registry credentials when deploying velero a
...
+```
+time="2023-11-06T10:53:21Z" level=error msg="Failed at copying to remote repository" Local PEID="ivd:fdc85e45-7c61-453f-9487-b7e47d020cd9:a6183167-d52d-4a83-932f-77104fac93e3" error="dial tcp: address 2620:124:6020:c304:0:a:0:763:443: too many colons in address" logSource="/go/src/github.com/vmware-tanzu/velero-plugin-for-vsphere/pkg/dataMover/data_mover.go:120"
+```
+The workaround to this problem is to use VC FQDN instead of IPV6 format in `csi-vsphere.conf` file. See the following example.
+
+```
+[Global]
+cluster-id = "cluster0"
+
+[VirtualCenter "sc2-04-dhcpv6-v2623-0763.ipv6.eng.vmware.com"]
```
Comment (xing-yang) [inline docs/known_issues.md:81]: Is this a real name?  Don't use a real name.

File: docs/supervisor-notes.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ Below is the networking diagram of velero-plugin-for-vsphere in vSphere with Tan
 
 Certain resources with the **vSphere with Tanzu** Supervisor Cluster need to be created by the Supervisor Cluster and cannot be restored. The **Velero Plugin for vSphere** blocks backup and restore of these resources and will generate errors if an attempt is made to backup or restore these resources and the backup or restore will be marked as "Partially Failed".
 
+Similarly, certain resources with the **Tanzu Kubernetes Grid** multicloud (TKGm) need to be created by the TKGm cluster and cannot be restored. The **Velero Plugin for vSphere** blocks backup and restore of these resources and will generate errors if an attempt is made to backup or restore these resources and the backup or restore will be marked as "Partially Failed".
```
Comment (xing-yang) [inline docs/supervisor-notes.md:19]: What are the resources that cannot be restored?  Are you saying the backup is successful but restore failed?

File: docs/known_issues.md (hunk 1 above)
Comment (lipingxue) [reply docs/known_issues.md]: Fixec.

File: docs/supervisor-notes.md (hunk 2 above)
Comment (lipingxue) [reply docs/supervisor-notes.md:19]: No. Both backup and restore will fail. In general, during backup and restore, if user does not specify "--exclude-resources"  to exclude the resource listed in the resource block list, backup/restore will failed with error like "Resource CRD XXX is blocked in backup, skipping".
We have already documented this for Supervisor cluster(line 16-18), and we observed the same behavior in TKGm, and that is why we added line 19 for TKGm.

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/849/

File: 1 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Address comments from Xing.
```
Comment (xing-yang) [inline 1:1]: I wonder why these files show up as submitted files to be checked in.

File: 2 [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+#
+# On branch lipingx-doc-update-July-24
+# Changes to be committed:
+#	modified:   docs/vanilla.md
+#
+# Untracked files:
+#	.DS_Store
+#	docs/.DS_Store
+#	set-unit-env.sh
+#
```
Comment (xing-yang) [inline 2:13]: No sure why this showed up.  You may want to rebase your branch.

File: set-unit-env.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export VDDK_LIBS=github.com/vmware-tanzu/velero-plugin-for-vsphere/.libs/vmware-vix-disklib-distrib/lib64
+export TARGETS=./pkg/...
+export TIMEOUT=300s
+export VERBOSE=
+export DISABLE_CACHE=
+export RUN_SINGLE_CASE=controller
+
+
```
Comment (xing-yang) [inline set-unit-env.sh:10]: Is this an intended change?

File: 2 (hunk 4 above)
Comment (lipingxue) [reply 2]: Removed this file

**Comment by svcbot-qecnsdp** [comment]
Sorry, some of the pre-check tests are failing for this PR ! 

Please check the Jenkins job for more details
https://container-dp.svc.eng.vmware.com/job/CNSDP-CI/850/

File: set-unit-env.sh (hunk 5 above)
Comment (lipingxue) [reply set-unit-env.sh]: Removed this file.

**Review by xing-yang — APPROVED** [review]
lgtm

**Pull request merged**